flake.nix

```diff
@@ -187,6 +188,39 @@
             branches = [ "develop" ];
             plugins = [{ "thedyrt/skip-checkout#v0.1.1" = null; }];
           };
+          runIntegrationTest = test:
+            { with-archive ? false }: {
+              command =
+                runInEnv self.devShells.x86_64-linux.integration-tests ''
+                  export GOOGLE_CLOUD_KEYFILE_JSON=$AUTOMATED_VALIDATION_SERVICE_ACCOUNT
+                  export GCLOUD_API_KEY=$(cat $INTEGRATION_TEST_LOGS_GCLOUD_API_KEY_PATH)
+                  source $INTEGRATION_TEST_CREDENTIALS
+                  export AWS_ACCESS_KEY_ID AWS_SECRET_ACCESS_KEY
+                  export KUBE_CONFIG_PATH=$$HOME/.kube/config
+                  gcloud auth activate-service-account --key-file=$AUTOMATED_VALIDATION_SERVICE_ACCOUNT automated-validation@o1labs-192920.iam.gserviceaccount.com --project o1labs-192920
```

Comment:
Won't these variables end up in the Nix store? If yes, that'd be a bad security posture. Perhaps even related to #10692.

Reply:
Nope, this is performed completely outside the store. What ends up in the pipeline is only the variable names, the contents are read at "runtime" during step execution.

Reply:
Ah, right. Makes sense!